jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -220,17 +223,114 @@
     suffix:
       - test-unit-integration: # kubernetes-pull-test-unit-integration
           status-context: unit/integration
+          max-total: 0 # Unlimited
           skip-if-no-test-files: false
+          success-comment: ''
+          failure-comment: ''
+          error-comment: ''
+          trigger-phrase: 'unit\s+test'
           cmd: |
             export KUBE_VERIFY_GIT_BRANCH="${{ghprbTargetBranch}}"
             ./hack/jenkins/gotest-dockerized.sh
       - verify-all: # kubernetes-pull-verify-all
           status-context: verification
+          max-total: 0 # Unlimited
           skip-if-no-test-files: true
+          success-comment: ''
+          failure-comment: ''
+          error-comment: ''
+          trigger-phrase: 'verify'
           cmd: |
             export KUBE_VERIFY_GIT_BRANCH="${{ghprbTargetBranch}}"
             export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
             ./hack/jenkins/gotest-dockerized.sh
+      - build-test-e2e-gce: # kubernetes-pull-build-test-e2e-gce
+          status-context: GCE e2e
+          max-total: 12
+          skip-if-no-test-files: false
```

Comment:
Will you add a comment (maybe in the job template?) explaining what this does and why we need it (especially true for verify-all)